docs/help/submit-issue.md

```diff
@@ -0,0 +1,108 @@
+
+# Submit an Issue
+
+## Where to Find Help
+
+When you run into issues while building or using the Trio app, follow the steps below to get help efficiently and show respect for the time our volunteers generously offer.
+
+-  [ ] Always start by searching [**TrioDocs**](https://triodocs.org) — just use the 🔎 search bar at the top.
+- [ ] Search your favorite support group (Facebook or Discord):  
+      
+    !!! tip "Help the Community Help You"
+        
+        When you run into issues building or using the _Trio_ app, there’s a **wonderful community of loopers** ready to help. These volunteers offer their time freely — no one is paid to support or troubleshoot your setup. They simply want to help others.  
+        To respect their efforts and get the most helpful response, please do your homework first and **include the key details they’ll need when asking for help**.
+      
+    - **[Facebook](https://facebook.triodocs.org)**
+        -  Not sure **how to search in** a **Facebook** group? [Here’s a video to help.](https://www.youtube.com/watch?v=_vSN6C-Uo04)  
+        - If you use **Facebook**, check the **Featured posts** at the top of the group, where many common questions have already been answered.
+    - **[Discord](https://discord.triodocs.org)**: if your issue is related to:
+        - [Build with Mac](https://discord.com/channels/1020905149037813862/1239982577008377958)
+        - [Build with Browser](https://discord.com/channels/1020905149037813862/1239982525820960860)
+        - [Trio Settings](https://discord.com/channels/1020905149037813862/1239982177190678549)  
+      
+- [ ] **Avoid posting the same question in multiple groups.**  
+    Many mentors follow several communities, and cross-posting can lead to confusion. Only try another group if you've received no replies after several hours.
```

Comment:
I would modify the several hours. Perhaps, "Even if you feel your situation cannot wait, give the mentors several hours to respond. Most questions get an answer the same day."

Reply:
✔️ Addressed in 436ca41